recipes.js

```diff
@@ -1,10 +1,86 @@
-const mongoose = require('mongoose');
-const Schema   = mongoose.Schema;
-const data = require('./data.js');
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const data = require("./data.js");
 
-mongoose.connect('mongodb://localhost/recipeApp')
+mongoose
+  .connect("mongodb://localhost/recipeApp")
   .then(() => {
-    console.log('Connected to Mongo!');
-  }).catch(err => {
-    console.error('Error connecting to mongo', err);
+    console.log("Connected to Mongo!");
+  })
+  .catch(err => {
+    console.error("Error connecting to mongo", err);
   });
+
+// Iteration 1: Create a Schema
+const recipeSchema = new Schema({
+  title: { type: String, required: true, unique: true },
+  level: {
+    type: String,
+    enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"]
+  },
+  ingredients: { type: Array },
+  cuisine: { type: String, required: true },
+  dishType: {
+    type: String,
+    enum: ["Breakfast", "Dish", "Snack", "Drink", "Dessert", "Other"]
+  },
+  image: {
+    type: String,
+    default: /https:\/\// + "images.media-allrecipes.com/images/75131.jpg"
```

Comment:
this doesn't give you exactly what you want - `/https:\/\//images.media-allrecipes.com/images/75131.jpg` is how it appears in the database.